src/main/java/org/grizz/exception/MaxNumberOfPlanetException.java

```diff
@@ -0,0 +1,5 @@
+package org.grizz.exception;
+
+public class MaxNumberOfPlanetException extends RuntimeException {
```

Comment:
I run out in the future. I don't know why I keep thinking about the functions that we will probably must implements in the feature.